intl_translation_arb/test/intl_translation_arb_test.dart

```diff
@@ -1,12 +1,40 @@
 import 'package:intl_translation_arb/intl_translation_arb.dart';
+import 'package:intl_translation_format/src/translation_format.dart';
 import 'package:intl_translation_format/test_utils.dart';
 import 'package:test/test.dart';
 
 void main() {
   group('Arb Format -', () {
-    testFormatParserWithDefaultMessages(
-      ArbFormat(),
-      simpleMessage: '''
+    testFormat(ArbDefaultFormatTester());
+
+    test('File name', () {
+      final template = MockTemplate('intl', {});
+      template.lastModified = null;
+      final result = ArbFormat().generateTemplateFiles(template);
+
+      expect(result.first.name, 'intl_en.arb');
+    });
+
+    test('Locale fr', () {
+      final template = MockTemplate('intl', {}, locale: 'fr');
+      template.lastModified = null;
+      final result = ArbFormat().generateTemplateFiles(template);
+
+      expect(result.first.name, 'intl_fr.arb');
+      expect(result.first.contents, '''
+{
```

Comment:
nit: the formatting here was confusing at first. Where's the opening triple quote? If you can put it on a separate line I'd do that, but if the formatter does this then I'd suggest splitting the expected result into a variable to make the triple-quoted string more obvious.